raigad/src/main/java/com/netflix/raigad/monitoring/NodeHealthMonitor.java

```diff
@@ -0,0 +1,105 @@
+/**
+ * Copyright 2014 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.netflix.raigad.monitoring;
+
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+import com.netflix.raigad.configuration.IConfiguration;
+import com.netflix.raigad.scheduler.SimpleTimer;
+import com.netflix.raigad.scheduler.Task;
+import com.netflix.raigad.scheduler.TaskTimer;
+import com.netflix.raigad.utils.ElasticsearchProcessMonitor;
+import com.netflix.servo.annotations.DataSourceType;
+import com.netflix.servo.annotations.Monitor;
+import com.netflix.servo.monitor.Monitors;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.concurrent.atomic.AtomicReference;
+
+/**
+ * Created by alfasi on 4/22/15.
+ */
+@Singleton
+public class NodeHealthMonitor extends Task {
+    private static final Logger logger = LoggerFactory.getLogger(NodeHealthMonitor.class);
+    public static final String METRIC_NAME = "Elasticsearch_NodeHealthMonitor";
+    private final ElasticsearchNodeHealthReporter healthReporter;
+
+    @Inject
+    public NodeHealthMonitor(IConfiguration config) {
+        super(config);
+        healthReporter = new ElasticsearchNodeHealthReporter();
+        Monitors.registerObject(healthReporter);
+    }
+
+    @Override
+    public void execute() throws Exception {
+
+        // If Elasticsearch is started then only start the monitoring
+        if (!ElasticsearchProcessMonitor.getWasElasticsearchStarted()) {
+            String exceptionMsg = "Elasticsearch is not yet started, check back again later";
+            logger.info(exceptionMsg);
+            return;
+        }
+
+        HealthBean healthBean = new HealthBean();
+        try {
+            healthBean.esprocessdown = 0;
+            if (!ElasticsearchProcessMonitor.isElasticsearchRunning()) {
+                healthBean.esprocessdown = 1;
+            }
```

Comment:
Add healthBean.esprocessdown = 0; to else statement. So that it gets executed only when ES process is Up. 

No need to set the value twice if ES is not Running.